dojo.py

```diff
@@ -0,0 +1,32 @@
+class Dojo():
+
+	def __init__(self):
+		self.rooms = []
+		self.offices = []
+		self.livingSpaces = []
+		self.fellows = []
+		self.staffs = []
+
+	def create_room(self, room_name):
+		""" function to create a unique room space """
+		room_name = room_name.lower()
+
+		if room_name in self.rooms:
+			return "Room already exists"
+		else:
+			room_type = input("Enter room type for " + room_name + ":")
```

Comment:
In your implementation of the logic make use of classes to create room objects with the attributes. Consider changing your implementation logic to make use of classes for the rooms.